### ISSUE TITLE
Toolbar toolchain button throws error after build button is pressed

### ISSUE THREAD

**Issue by cmarincia** [xmake-io/xmake-vscode#166]
### Xmake Version

v2.7.5+202301262306

### Operating System Version and Architecture

Linux catalin 6.1.5-229.current #1 SMP PREEMPT_DYNAMIC Thu Jan 12 20:29:03 UTC 2023 x86_64 GNU/Linux

### Describe Bug

I can build using this xmake.lua
![image](https://user-images.githubusercontent.com/35623921/215233739-c39fbf98-fd29-41e1-a2a1-7edbe5c53c90.png)

When I click the toolchain button it throws an error.
![image](https://user-images.githubusercontent.com/35623921/215233769-0e13d7a7-8fef-48e7-b347-ec00c6c83452.png)

Restarting VSCode does fix the problem until I press the build button again.

### Expected Behavior

I should be able to select a new toolchain to build the project with.

### Project Configuration

```
add_rules("mode.debug", "mode.release")
set_languages("c99")
target("clox")
    set_kind("binary")
    add_headerfiles("src/*.h")
    add_files("src/*.c")
```

**Comment by waruqi** [comment]
I can not replicate it.

**Comment by waruqi** [comment]
I have fixed it in 1.9.4
